packages/node/src/yargs.ts

```diff
@@ -152,6 +152,12 @@ export const yargsOptions = yargs(hideBin(process.argv))
       describe: 'Specify the dictionary api for this network',
       type: 'string',
     },
+    'auth-dictionary': {
+      demandOption: false,
+      describe: 'Use auth center as endpoint for dictionary',
```

Comment:
It would be good to have a better name and description.

Reply:
updated